src/main/resources/finalCopy/final_workflow_log_dir.test

```diff
@@ -0,0 +1,11 @@
+# Tests that the runtime option to preserve logs from the workflow is being honored.
+
+name: workflow_log_dir
+testFormat: workflowsuccess
+backends: [Local]
+tags: [localdockertest]
```

Comment:
Presumably this is local only because it's awkward to verify it got copied correctly in GCS?